.github/workflows/linux.yml

```diff
@@ -349,11 +200,12 @@ jobs:
           python -m pip wheel -v --no-deps --wheel-dir ${BUILD_DIR} \
               --config-settings=override=cross.arch="manylinux_2_31_x86_64" \
               --config-settings=override=cmake.options.BUILD_FAST_TOKENIZERS="${{ matrix.build_fast_tokenizers }}" \
-              --find-links ${INSTALL_DIR}/tools \
+              ${{ needs.openvino_download.outputs.ov_wheel_source }} \
               ${OPENVINO_TOKENIZERS_REPO}
         env:
           CMAKE_BUILD_PARALLEL_LEVEL: '4'
           CMAKE_GENERATOR: 'Unix Makefiles'
+        working-directory: ${{ env.INSTALL_DIR }}
```

Comment:
Do we need change cwd because ${{ needs.openvino_download.outputs.ov_wheel_source }} is a relative path?

Reply:
Right, 'cause we don't know the install directory in advance when we unpack artifacts downloaded from GitHub